src/test/java/de/tum/in/www1/artemis/ExamIntegrationTest.java

```diff
@@ -1350,14 +1348,10 @@ public void unlockAllRepositories() throws Exception {
         var participationEx2St1 = database.addStudentParticipationForProgrammingExercise(programmingExercise2, "student1");
         var participationEx2St2 = database.addStudentParticipationForProgrammingExercise(programmingExercise2, "student2");
 
-        assertThat(studentExamRepository.findStudentExam(programmingExercise, participationExSt1)).isPresent();
-        assertThat(studentExamRepository.findStudentExam(programmingExercise, participationExSt1).get()).isEqualTo(studentExam1);
-        assertThat(studentExamRepository.findStudentExam(programmingExercise, participationExSt2)).isPresent();
-        assertThat(studentExamRepository.findStudentExam(programmingExercise, participationExSt2).get()).isEqualTo(studentExam2);
-        assertThat(studentExamRepository.findStudentExam(programmingExercise2, participationEx2St1)).isPresent();
-        assertThat(studentExamRepository.findStudentExam(programmingExercise2, participationEx2St1).get()).isEqualTo(studentExam1);
-        assertThat(studentExamRepository.findStudentExam(programmingExercise2, participationEx2St2)).isPresent();
-        assertThat(studentExamRepository.findStudentExam(programmingExercise2, participationEx2St2).get()).isEqualTo(studentExam2);
+        assertThat(studentExamRepository.findStudentExam(programmingExercise, participationExSt1)).isPresent().contains(studentExam1);
+        assertThat(studentExamRepository.findStudentExam(programmingExercise, participationExSt2)).isPresent().contains(studentExam2);
+        assertThat(studentExamRepository.findStudentExam(programmingExercise2, participationEx2St1)).isPresent().contains(studentExam1);
+        assertThat(studentExamRepository.findStudentExam(programmingExercise2, participationEx2St2)).isPresent().contains(studentExam2);
```

Comment:
```suggestion
        assertThat(studentExamRepository.findStudentExam(programmingExercise, participationExSt1)).contains(studentExam1);
        assertThat(studentExamRepository.findStudentExam(programmingExercise, participationExSt2)).contains(studentExam2);
        assertThat(studentExamRepository.findStudentExam(programmingExercise2, participationEx2St1)).contains(studentExam1);
        assertThat(studentExamRepository.findStudentExam(programmingExercise2, participationEx2St2)).contains(studentExam2);
```
Is the additional `isPresent` necessary in this case? I assume that `contains` already checks that the optional is not empty and provides good error messages in case it is.